.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Monthly, 1st of the month, 6am
+    - cron: "0 6 1 * *"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
Can we drop 3.7?

Reply:
We can!  Though it is still officially supported by python.org, though that will change in mid-2023.

Reply:
I think most packages in the python scientific stack have adopted NEP 29 (https://numpy.org/neps/nep-0029-deprecation_policy.html).

Reply:
That makes sense.  Given that I have no plans to add features or anything to this package, only to fix the build/install and verify it via CI, then I'm happy to have a wider allowed version schedule.  Especially given that whatever code still uses it is probably also old.  Of course if it stops working on Python 3.7 or Numpy 1.19 next year, happy to immediately drop support.  I just didn't want to artificially bump allowed versions of dependencies without having a good reason for it.

Reply:
I think dropping Python 3.7 can be a separate PR if needed, especially given it is still green here.